### PR TITLE
Fixes: Concerto and Transform upgrades + allows markdown_cicero format

### PR DESCRIPTION
**Pull request by jeromesimeon**
# Issue #574 & more

### Changes
- Upgrade to Concerto `0.82.8`
- Upgrade to Markdown Transform `0.12.2` (Fixes critical issue with Ergo formulas #572 )
- Upgrade to Ergo `0.21.1`
- Allows `markdown_cicero` as proper format

### CODE REVIEW

File: packages/cicero-core/test/clause.js [hunk 1]
```diff
@@ -601,6 +573,16 @@ Assignment. Licensee may freely assign or otherwise transfer all or any of its r
             newText.should.eql(text);
         });
 
+        it('should be able to draft a copyright license (markdown_cicero)', async function() {
+            const template = await Template.fromDirectory('./test/data/copyright-license', options);
+            const clause = new Clause(template);
+            const data = copyrightData;
+            clause.setData(data);
+            const newText = clause.draft({format:'markdown_cicero'});
+            // remove the generated clause id
```
Comment (DianaLease) [inline packages/cicero-core/test/clause.js:582]: I don't understand this comment "`remove the generated clause id`"

Comment (jeromesimeon) [reply packages/cicero-core/test/clause.js:582]: yes it makes no sense. cut-paste from previous test which had a non-sensical comment!

Comment (jeromesimeon) [reply packages/cicero-core/test/clause.js:582]: fixed!

Comment (jeromesimeon) [reply packages/cicero-core/test/clause.js]: fixed now...

**Pull request merged**